src/infra/sequelize/projections/modificationRequest/updates/abandon/onLegacyAbandonSupprimé.ts

```diff
@@ -0,0 +1,14 @@
+import { logger } from '@core/utils'
+import { LegacyAbandonSupprimé } from '@modules/project'
+
+export const onLegacyAbandonSupprimé =
+  (models) =>
+  async ({ payload }: LegacyAbandonSupprimé) => {
+    const { projetId } = payload
+    try {
+      const ModificationRequestModel = models.ModificationRequest
+      await ModificationRequestModel.destroy({ where: { projectId: projetId, type: 'abandon' } })
```

Comment:
Tu es obligée d'en faire une constante ici ? 
Pourquoi pas simplement : 

```suggestion
      await models.ModificationRequest.destroy({ where: { projectId: projetId, type: 'abandon' } })
```